.buildkite/pipeline.yml

```diff
@@ -21,6 +21,10 @@ x-common-params:
         # They are passed from the Buildkite agent to the Docker container
         - 'AWS_ACCESS_KEY'
         - 'AWS_SECRET_KEY'
+  - &nvm_plugin
+    automattic/nvm#5e32ba05632164c68b0c8bec52c41dd344068fb2
```

Comment:
Just waiting for https://github.com/Automattic/nvm-buildkite-plugin/pull/11 version 0.2.1 to ship, then this should be ready to merge